src/cdk-experimental/menu/menu-bar.ts

```diff
@@ -157,73 +127,35 @@ export class CdkMenuBar extends CdkMenuGroup implements Menu, AfterContentInit,
     }
   }
 
-  /** Setup the FocusKeyManager with the correct orientation for the menu bar. */
-  private _setKeyManager() {
-    this._keyManager = new FocusKeyManager(this._allItems)
-      .withWrap()
-      .withTypeAhead()
-      .withHomeAndEnd();
-
-    if (this._isHorizontal()) {
-      this._keyManager.withHorizontalOrientation(this._dir?.value || 'ltr');
-    } else {
-      this._keyManager.withVerticalOrientation();
-    }
-  }
-
   /**
    * Set the PointerFocusTracker and ensure that when mouse focus changes the key manager is updated
    * with the latest menu item under mouse focus.
    */
   private _subscribeToMouseManager() {
     this._ngZone.runOutsideAngular(() => {
-      this._pointerTracker = new PointerFocusTracker(this._allItems);
-      this._pointerTracker.entered.pipe(takeUntil(this._destroyed)).subscribe(item => {
-        if (this._hasOpenSubmenu()) {
-          this._keyManager.setActiveItem(item);
+      this.pointerTracker = new PointerFocusTracker(this.items);
+      this.pointerTracker.entered.pipe(takeUntil(this.destroyed)).subscribe(item => {
+        // TODO(mmalerba): Why does the menu bar do this in a hasOpenSubmenu check
```

Comment:
Calling out this TODO in case anyone has an idea why menu is different from menubar in this way

Reply:
Ok I think I understand why this is different. In drive when you arrow between the "file", "edit", etc. menus in the menu bar it automatically pops them open, but sub-menus are not automatically opened you have to press right arrow / space / enter to expand them

Reply:
Ok this makes sense. Its because we don't want to start auto-focusing different menu bar items on hover unless the menu has already been opened.